.editorconfig

```
@@ -0,0 +1,15 @@
+# http://editorconfig.org
+root = true
+
+[*]
```

Maybe only for certain files, instead of for all files. This is potential very unwanted..
```suggestion
[*.{ts,js,json,yml}]
```

---

Seems like a reasonable suggestion.  But I would also argue that the config that this repo provides is just a baseline.  If someones uses this utility to start a new project they can add/change configuration to meet their needs if they are authoring other filetypes and for some reason don't want this config.  My two cents.

Lastly, if this line 4 changes, then the lower line 13 configuring *.md needs to be updated to include more configuration from this first section.

@bitjson thoughts?

---

Thanks for really thinking about it!

I think it's important that this project continues to feel minimal by default, and only includes the extras when the user enables them.

If we have this option off by default (like the Travis CI config), including the more specific `.editorconfig` could be a good thing. Without seeing a full RegEx example (`[*.{ts,js,json,yml}]`), it might not be as intuitively obvious how the configuration works. So specifying the file types might be a bit more helpful in that respect. (I don't have a strong opinion about that though.)